### ISSUE TITLE
how to use the trained model to test my own dataset

### ISSUE THREAD

**Issue by maggiequan** [princeton-vl/CornerNet#56]
hello,i have huge pictures and wanna to use the trained model to test ,how can i do it?

**Comment by woaiwoziji** [comment]
Have you solved this? I found it has been limited to the coco dataset

**Comment by djmirv** [comment]
问题解决了吗？Ever solved this?

**Comment by maggiequan** [comment]
@djmirv @woaiwoziji  已经解决了，我把过程记录在了csdn博客上，可以自己去看看，博客名就是github的名字

**Comment by yx-wu** [comment]
I tested my data set with CornerNet and got the result. json file, which is roughly as follows. Is there no post-processing part in the code? Why do I get a lot of test results for each test picture? And the four values in bbox in result.json.  I got are x-min, y-min, width and height (because some results have very small numbers corresponding to witdh or heigh)? 
[{"image_id": 5501, "category_id": 1, "bbox": [261.49, 301.42, 248.6, 208.57], "score": 0.23}, {"image_id": 5501, "category_id": 1, "bbox": [0.01, 508.54, 510.08, 1.45], "score": 0.22}, {"image_id": 5501, "category_id": 1, "bbox": [492.72, 476.82, 17.37, 33.17], "score": 0.22}, {"image_id": 5501, "category_id": 1, "bbox": [189.36, 480.88, 320.72, 29.11], "score": 0.21}, {"image_id": 5501, "category_id": 1, "bbox": [500.68, 305.13, 9.4, 204.86], "score": 0.2},
